src/user-management/interfaces/refresh-and-seal-session-data.interface.ts

```diff
@@ -18,18 +19,35 @@ export enum RefreshAndSealSessionDataFailureReason {
   ORGANIZATION_NOT_AUTHORIZED = 'organization_not_authorized',
 }
 
-// TODO: These should be renamed since it's possible to have an unsealed session
-type RefreshAndSealSessionDataFailedResponse = {
+type RefreshSessionFailedResponse = {
   authenticated: false;
   reason: RefreshAndSealSessionDataFailureReason;
 };
 
+/**
+ * @deprecated To be removed in a future major version along with `refreshAndSealSessionData`.
+ */
```

Comment:
These types are all a bit of a mess because we require a major version bump to clean up the callsites.